lib/nuget-parser/index.ts

```diff
@@ -58,13 +63,25 @@ export async function buildDepTreeFromFiles(
     version: '0.0.0',
   };
 
-  let targetFramework;
+  let targetFramework: TargetFramework | undefined;
   try {
     if (manifestType === 'dotnet-core') {
       targetFramework = await getTargetFrameworksFromProjFile(projectRootFolder);
     } else {
       // .csproj is in the same directory as packages.config or project.json
       targetFramework = await getTargetFrameworksFromProjFile(path.resolve(fileContentPath, '../'));
+
+      // finally, for the .NETFramework project, try to assume the framework using dotnet-deps-parser
+      if (!targetFramework) {
+        const extractedFrameworks = await depsParser.extractTargetFrameworksFromFiles(safeRoot, safeTargetFile);
```

Comment:
`extractTargetFrameworksFromFiles` is intened for test only, please use the relevant function directly for each type of file you are parsing. 

So in this case you are adding ability to process package.config alone so you can use: 
```
if (_.endsWith(manifestFilePath, 'packages.config')) {
    return extractTargetFrameworksFromProjectConfig(manifestFileContents);

```
verify file name ends with packages.config then pass it in after you read it in from disk

Reply:
Good suggestion, added